.travis.yml

```diff
@@ -7,6 +7,7 @@ sudo: false
 branches:
   only:
     - master
+    - isim/intg-test-script
```

Comment:
To be removed once testing is done.

Reply:
Just a nit; `.travis.yml` will only be used when this branch gets merged, no?

Reply:
Ivan clarified. This is for having Travis actually run this branch :+1: